sdcm/remote/remote_file.py

```diff
@@ -23,12 +23,29 @@
 
 
 LOGGER = logging.getLogger(__name__)
+# The below set of SECURED_PARAMETERS should include relevant password-parameter-names,
+# to be masked by asterisks when printed to log.
+SECURED_PARAMETERS = {'ldap_bind_passwd'}
 
 
 def read_to_stringio(fobj):
     return StringIO(fobj.read())
 
 
+def secured_masked_content(parameters_content: str) -> str:
```

Comment:
What does parameters mean in the input of this function ?

Reply:
it aims to make it clearer to understand it's not a plain string, but a string that could be yaml-loaded to a form of a dictionary.
It will only mask by parameter names received in such a format.

Reply:
it's not SCT params, it's aiming for scylla.yaml, I can understand why it would be call `content` or `yaml_string`, but what `parameters_content` tells me ?

also I think it should be 'mask_secured_content` to be a bit more clear what it's doing

Reply:
@fruch , does `yaml_content` sound better?

Reply:
yes